src/PowerShellEditorServices/Workspace/ScriptFileMarker.cs

```diff
@@ -162,36 +164,27 @@ internal static ScriptFileMarker FromDiagnosticRecord(PSObject psObject)
                 };
             }
 
+            string severity = diagnosticRecord.Severity.ToString();
+            if(!Enum.TryParse(severity, out ScriptFileMarkerLevel level))
```

Comment:
Minor nit, space between `if` and opening `(`.

Reply:
I'm glad I wasn't the only one that felt like this

Reply:
Good catch - fixed.